### ISSUE TITLE
Uncaught TypeError: Cannot read property 'abortKeyBinding' of undefined

### ISSUE THREAD

**Issue by NickBogdanov** [ianobermiller/split-string-on-enter#1]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.5.3
**System**: Mac OS X 10.11.2
**Thrown From**: [split-string-on-enter](https://github.com/ianobermiller/split-string-on-enter) package, v1.0.1
### Stack Trace

Uncaught TypeError: Cannot read property 'abortKeyBinding' of undefined

```
At /Users/nick/.atom/packages/split-string-on-enter/lib/split-string-on-enter.js:68

TypeError: Cannot read property 'abortKeyBinding' of undefined
    at Object.splitString (/Users/nick/.atom/packages/split-string-on-enter/lib/split-string-on-enter.js:68:12)
    at atom-workspace.<anonymous> (/Users/nick/.atom/packages/split-string-on-enter/lib/split-string-on-enter.js:30:19)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Users/nick/Downloads/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at CommandRegistry.handleCommandEvent (/Users/nick/Downloads/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Users/nick/Downloads/Atom.app/Contents/Resources/app.asar/src/command-registry.js:160:19)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchApplicationMenuCommand (/Users/nick/Downloads/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:1013:28)
    at emitOne (events.js:77:13)
    at EventEmitter.emit (events.js:169:7)
```
### Commands

```
     -2:05.6.0 core:save (atom-text-editor.editor.is-focused)
     -2:05.5.0 linter:lint (atom-text-editor.editor.is-focused)
     -1:59.4.0 typescript:format-code (atom-text-editor.editor.is-focused)
     -1:58.2.0 core:save (atom-text-editor.editor.is-focused)
     -1:58.1.0 linter:lint (atom-text-editor.editor.is-focused)
     -0:37.3.0 core:select-all (atom-text-editor.editor.mini.is-focused)
     -0:37 core:paste (atom-text-editor.editor.mini.is-focused)
     -0:36.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:19.4.0 linter:lint (atom-text-editor.editor)
  4x -0:15.7.0 core:move-left (atom-text-editor.editor.is-focused)
     -0:14.9.0 split-string-on-enter:split (atom-text-editor.editor.is-focused)
     -0:14.9.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:13.7.0 core:undo (atom-text-editor.editor.is-focused)
     -0:13.2.0 core:save (atom-text-editor.editor.is-focused)
     -0:13.1.0 linter:lint (atom-text-editor.editor.is-focused)
     -0:01.5.0 split-string-on-enter:split (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "monokai"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
angularjs, v0.3.4
atom-jasmine, v0.8.1
atom-typescript, v8.3.0
browser-plus, v0.0.54
console-log, v0.10.5
emmet, v2.4.1
file-icons, v1.6.16
linter, v1.11.3
linter-tslint, v0.8.1
local-history, v3.2.3
monokai, v0.18.0
split-string-on-enter, v1.0.1

# Dev
No dev packages
```

**Comment by Daniel15** [comment]
I see this error when I press enter in the Nuclide auth dialog (the one that prompts for two-factor auth):

```
TypeError: Cannot read property 'abortKeyBinding' of undefined
  at Object.splitString (C:\Users\danlo\.atom\packages\split-string-on-enter\lib\split-string-on-enter.js:41:12)
  at atom-workspace.<anonymous> (C:\Users\danlo\.atom\packages\split-string-on-enter\lib\split-string-on-enter.js:30:20)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\danlo\AppData\Local\atom\app-1.7.2\resources\app.asar\src\command-registry.js:260:29)
  at CommandRegistry.handleCommandEvent (C:\Users\danlo\AppData\Local\atom\app-1.7.2\resources\app.asar\src\command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (C:\Users\danlo\AppData\Local\atom\app-1.7.2\resources\app.asar\src\command-registry.js:160:19)
  at AtomEnvironment.module.exports.AtomEnvironment.dispatchApplicationMenuCommand (C:\Users\danlo\AppData\Local\atom\app-1.7.2\resources\app.asar\src\atom-environment.js:1100:28)
  at EventEmitter.outerCallback (C:\Users\danlo\AppData\Local\atom\app-1.7.2\resources\app.asar\src\application-delegate.js:314:25)
  at emitTwo (events.js:87:13)
  at EventEmitter.emit (events.js:172:7)
```

**Comment by ianobermiller** [comment]
Fix deployed in v1.0.2

**Comment by Daniel15** [comment]
Thanks Ian, I'm not seeing this error any more :)
